ReactWindows/ReactNative.Shared/DevSupport/DevInternalSettings.cs

```diff
@@ -1,6 +1,10 @@
 ﻿using ReactNative.Modules.DevSupport;
 using System.Collections.Generic;
+#if WINDOWS_UWP
 using Windows.Storage;
+#else
+using PCLStorage;
+#endif
```

Comment:
Not seeing where this namespace is being used (since you're using the Settings.settings to read/write values)